examples/kinematics/average_angular_velocity_during_dive.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python3
+
+from sympy import solve, symbols, pi
+from symplyphysics import units, print_expression
+from symplyphysics.laws.kinematic import angular_frequency_from_radians_per_time as angular_velocity_law
+from symplyphysics.laws.kinematic import constant_acceleration_movement_is_parabolic as const_acceleration_law
+
+# Description
+## A diver makes 2.5 revolutions on the way from a 10-m-high platform to the water. Assuming zero initial
+## vertical velocity, find the average angular velocity during the dive.
+
+total_angular_displacement = symbols("total_angular_displacement")
+height = symbols("height")
+
+const_acceleration_law_sub = const_acceleration_law.law.subs({
+    const_acceleration_law.constant_acceleration: units.acceleration_due_to_gravity,
+    const_acceleration_law.initial_velocity: 0,
+    const_acceleration_law.distance(const_acceleration_law.movement_time): height,
+})
+# First solution is negative
+dive_time = solve(const_acceleration_law_sub, const_acceleration_law.movement_time)[1]
+
+angular_velocity_law_sub = angular_velocity_law.law.subs({
+    angular_velocity_law.radians: total_angular_displacement,
+    angular_velocity_law.time: dive_time,
+})
+average_angular_velocity = solve(
```

Comment:
Add explanation, why frequency is an average velocity.

Reply:
Are angular frequency and angular frequency not synonyms? Although they do sound as if they are not

Reply:
It seems they are. Although frequency is absolute value of velocity. Also taking velocity for average velocity only works when it is constant.

Reply:
First, in this example we are not interested in the sign of the velocity, only in its absolute value, so to me it is quite reasonable to for them to be interchangeable here. Next, the usual formula for finding average velocity is average displacement over change in time ($`\overline{v} = \frac {\Delta x} {\Delta t}`$), and here I take the same approach: $`\overline{\omega} = \frac {\Delta \theta} {\Delta t}`$.

Reply:
Yes but I want to see it in comments in the code, not in github review.

Reply:
Oh, I see what you mean